qa/automation/nikeev/Any Career Portal Page/Cookies.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Cookies settings on any career portal page
+
+Background: 'Necessary cookies' is a required option in Cookies Settings box
+
+Scenario Outline: User sees cookies pop-up on any portal page
+	Given user previously didn't accept cookies for career portal
+	When user opens <name of the page> page
```

Comment:
Better- When user clicks on <name> button

But you should mention somewhere what site you are testing and where these buttons are located